.github/workflows/grafana-bench.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     # Only run on push to the main branch
     branches: [main, master]
+  pull_request:
```

Comment:
I'm not sure if I see this as something that needs to run on every PR, but I'm happy to be shown  otherwise! What are your thoughts?

Reply:
If we don't run on a PR then the only chance to debug it is on merge which is tedious, especially if I need a review. I want to make sure tests are passing as often as possible (these were committed broken) and give feedback early and often.

Reply:
There's not too many PRs on this repo, I agree it would be good to validate early. Perhaps conditionally, as long as the simpler unit tests pass